Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml

```diff
@@ -56,4 +56,27 @@
     <!-- XTXT: not in AllowList dialog message -->
     <string name="dcc_ticketing_not_in_allow_list_dialog_message">Der Anbieter ist uns nicht bekannt. Bitte überprüfen Sie, ob der QR-Code wirklich von dem Anbieter stammt und ob Sie ihm Ihr Zertifikat zur Verfügung stellen wollen.</string>
```

Comment:
Not part of this PR, but this sentence should be updated. It was changed since it has been implemented. Updated version: "Der Anbieter ist uns nicht bekannt. Aus Sicherheitsgründen ist die Überprüfung bei Ticketbuchung nur für bekannte Anbieter möglich. Bitte kontaktieren Sie den Anbieter, der ihnen dieses QR-Code bereitgstellt hat."